scripts/ci/publish.ts

```diff
@@ -893,10 +882,16 @@ async function testPackages(
     const pkg = packages[pkgName]
     if (pkg.packageJson.scripts.test) {
       console.log(`\nTesting ${chalk.magentaBright(pkg.name)}`)
-      if (process.env.BUILDKITE_PARALLEL_JOB === '2') {
+      // Sets ENV to override engines
+      if (process.env.BUILDKITE_PARALLEL_JOB === '0') {
```

Comment:
Why did we usually test all packages but not the client in step 0 (previously)?

Reply:
Think it was an attempt to speed up the buildkite tests

Reply:
I think we _extracted_ the client into its own job (hence we had job 0, 1 and 2) to speed things up indeed.